### DISCUSSION TITLE
9.2 -> 9.3 upgrade error

### DISCUSSION THREAD

**Discussion by StephenCovalli** [apache/skywalking#11241] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-booster-ui)

### What happened

When deploying skywalking-9.3.0 version, I noticed that hosting the UI page on a CentOS 6 system (kernel version 2.6.32-754.35.1.el6.x86_64) resulted in a long white screen when accessing it through a browser. Opening the console revealed an error message "net::err_content_length_mismatch." However, when deploying this version of UI on a CentOS 7 system (kernel version 3.10.0-1160.81.1.el7.x86_64), this issue did not occur. I noticed that there is a new reference to "brotli" in the iteration from version 9.2.0 to 9.3.0. Brotli is a compression algorithm, and I suspect that it may be causing the page loading error. Lastly, I would like to clarify that neither of my two servers has used Nginx or enabled its compression module.

### What you expected to happen

![image](https://github.com/apache/skywalking/assets/39046598/a00814f3-2eab-49ab-946f-e52e96aff64c)

### How to reproduce

Deploying skywalking-9.3.0 version on CentOS 6 (kernel version 2.6.32-754.35.1.el6.x86_64) and CentOS 7 (kernel version 3.10.0-1160.81.1.el7.x86_64).

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
We don't have experience about Nignx involved in our deployments. But these errors didn't occur in our deployment. 
Did you use a clean environment to deploy all? Including a new database? We don't provide a smooth upgrade guarantee.

**Reply by wu-sheng** [reply]
And OAP and UI are running on JVM, we don't care much about your Linux version.

**Reply by StephenCovalli** [reply]
I used default settings for the deployment, and the database was set to use H2 as the default option.

**Reply by zhoushuai1119** [reply]
Have you solved this problem? Can you tell me how to solve it?
